routes.py

```diff
@@ -66,8 +66,9 @@ def getalias():
     else:
         return {'status': 'invalid access'}, 401
 
-@obscura.route('/getlevel/<alias>', methods = ['GET', 'POST'])
-def getlevel(alias):
+
+@obscura.route('/level/<alias>', methods = ['GET', 'POST'])
```

Comment:
POST method is not required here.

Reply:
its for posting the ans

Reply:
Then GET is not required

Reply:
get is for getting the level on the basis of Alias parameter :P

Reply:
Cool